aws-out-of-cluster.md

```diff
@@ -32,6 +32,23 @@ More on AWS tagging [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/U
 
 Tags may take several hours to show up in the *Cost Allocations Tags* section described in the next step.
 
+### Custom tags mapping
+
+Tags that contain `:` in the key may be converted to `_` in the Kubecost UI due to Prometheus readability. To use AWS Label Mapping Configs, use this mapping format:
+
+```
+        helm:
```

Comment:
@bstuder99 Any reason you're formatting helm configs this way?

Reply:
Can you elaborate on the formatting?

Reply:
@bstuder99 I would've expected the code block to look like this, since this is usually how we show Helm Configs in our docs

```yaml
kubecostProdutConfigs:
  labelMappingConfigs:
    enabled: true
    namespace_external_label: mycompanybilling_namespace
    cluster_external_label: mycompanybilling_cluster
```

Similar to how it is configured here: https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.102/cost-analyzer/values.yaml#L951-L970

Reply:
Oh I see. Yeah, this is just copied from the corresponding issue, but let's do this formatting instead